src/main/java/game/Game.java

```diff
@@ -0,0 +1,152 @@
+package game;
+
+import game.field.Field;
+import org.apache.commons.lang.ArrayUtils;
+
+import java.util.Arrays;
+
+public class Game {
+
+    private PlayerController playerController;
+    private GUIController guiController;
+    private DiceController diceController;
+    private FieldController fieldController;
+    //private ChanceCardController chanceCardController;
+    private Player[] players;
+    private int playerTurn;
+
+    public static void main(String[] arg){
+        Game game = new Game();
+        game.gameLoop();
+    }
+
+    public Game(){
+        Field[] fields = Utility.fieldGenerator("src/main/java/resources/fieldList.xml");
+        guiController = new GUIController(fields);
+        diceController = new DiceController(2,6);
+        playerController = new PlayerController(guiController.returnPlayerNames(), 30000);
+        players = playerController.getPlayers();
+        playerTurn = (int) (Math.random() * (players.length - 1));
+    }
+
+    public void gameLoop(){
+        boolean stop = false;
+        boolean isDieIdentical;
+
+        guiController.addPlayers(players);
+
+        while(!stop){
+            playerTurn = getNextPlayerTurn();
+            guiController.showMessage("Player "+ playerController.getName(playerTurn) + " turn to roll the dice" );
+            // cast dice from dice controller
+            guiController.getUserButton("Roll the dice", "Roll");
+            diceController.roll();
+            isDieIdentical = diceController.isIdentical();
+            int dice1 = diceController.getFaceValue(0);
+            int dice2 = diceController.getFaceValue(1);
+
+            guiController.setDiceGui(dice1, (int) (Math.random() * 360), dice2, ((int) (Math.random() * 360)));
+
+            movePlayer(playerTurn, dice1+dice2);
+            while(isDieIdentical){
+                guiController.showMessage("You got identical dies, roll the dice again!");
+                guiController.getUserButton("Roll the dice", "Roll");
+                diceController.roll();
+                isDieIdentical = diceController.isIdentical();
+                dice1 = diceController.getFaceValue(0);
+                dice2 = diceController.getFaceValue(1);
+                guiController.setDiceGui(dice1, (int) (Math.random() * 360), dice2, ((int) (Math.random() * 360)));
+                movePlayer(playerTurn, dice1+dice2);
+            }
+            String userBtn = guiController.getUserButton("Continue or close game?",
+                    "Close game", "Continuegame");
+            if (userBtn.equals("Close game")) {
+                stop = true;
+                guiController.close();
+            }
```

Comment:
Skal det her ikke være uden for `while(!stop)` loopet?

Kan godt være at det bare er mig der er blind